Application/EdFi.Ods.Api/Security/Authorization/Repositories/RepositoryOperationAuthorizationDecoratorBase.cs

```diff
@@ -357,6 +374,7 @@ string BuildExistenceCheckSql(AuthorizationStrategyFilterResults[] resultsWithPe
                                 s.Append(')');
                             }, s);
 
+                        if(queryOptimized)
                         s.Append(')');
```

Comment:
You may want to consider adding some curly brackets for readability and to prevent a future developer from breaking this logic with a copy/paste error.

Reply:
Fixed it .Thanks